### PR TITLE
CPLP - 467 - Feature/partner network pagination

### PR DESCRIPTION
**Pull request by fatihayyildiz**
Partner network pagination and detail overlay component implemented
Modal component small alignment implemented for shared library

### CODE REVIEW

File: cx-portal/src/assets/locales/en/main.json [hunk 1]
```diff
@@ -131,7 +130,11 @@
       "bpnsearchlabel": "BPN Nummer",
```
Comment (oyo) [inline cx-portal/src/assets/locales/en/main.json:130]: not your PR - but there are german texts in english lang resources

File: cx-portal/src/assets/locales/en/main.json [hunk 2]
```diff
@@ -131,7 +130,11 @@
...
       "companynamesearchplaceholder": "Please type company name",
       "companynamesearchlabel": "Name der Firma",
```
Comment (oyo) [inline cx-portal/src/assets/locales/en/main.json:132]: another german text

**Review by oyo — APPROVED** [review]
lgtm :)  - but maybe you can fix those two german labels in the en-file

File: cx-portal/src/assets/locales/en/main.json (hunk 2 above)
Comment (fatihayyildiz) [reply cx-portal/src/assets/locales/en/main.json]: I updated the PR

Comment (oyo) [reply cx-portal/src/assets/locales/en/main.json]: thanks - then for me it's ready to merge :)

**Comment by fatihayyildiz** [comment]
I commit some other changes regarding the comment from Julia. Could you please take a quick look ? or is it ok to merge directly ? Those new commit contains small alignments

**Review by oyo — APPROVED** [review]
Hey Fatih - everything looks good to me now

**Pull request merged**